_episodes/01-starting-with-data.md

```diff
@@ -208,22 +209,29 @@ articles_df
 
 which prints contents like above.
 
-## Manipulating Our Species Survey Data
+## Manipulating our Articles data
 
 Now we can start manipulating our data. First, let's check the data type of the
-data stored in *articles_df* using the *type* method. The *type* method and
-*__class__* attribute tell us that *articles_df* is *<class 'pandas.core.frame.DataFrame'>*.
+data stored in `articles_df` using the `type` method. The `type` method and
+`__class__` attribute tell us that `articles_df` type is `<class 'pandas.core.frame.DataFrame'>`.
 
 ~~~
 type(articles_df)
 # this does the same thing as the above!
 articles_df.__class__
 ~~~
 {: .source}
+~~~
```

Comment:
there needs to bee empty line between those two source code block, otherwise they do not render properly